serializer/samples/simplesample_mqtt/simplesample_mqtt.c

```diff
@@ -174,7 +175,7 @@ void simplesample_mqtt_run(void)
             {
 #ifdef SET_TRUSTED_CERT_IN_SAMPLES
                 // For mbed add the certificate information
-                if (IoTHubClient_LL_SetOption(iotHubClientHandle, "TrustedCerts", certificates) != IOTHUB_CLIENT_OK)
+                if (IoTHubClient_LL_SetOption(iotHubClientHandle, OPTION_TRUSTED_CERT, certificates) != IOTHUB_CLIENT_OK)
```

Comment:
Cool!  For the blog, I can say: Device SDK samples include the option for trusted cert as an example?

Reply:
yep, if you would like to see more examples let me know.

---
In reply to: [140866383](https://github.com/Azure/azure-iot-sdk-c/pull/241#discussion_r140866383) [](ancestors = 140866383)

Reply:
Good for now!